addons/udes_stock/models/stock_picking_batch.py

```diff
@@ -1268,6 +1272,37 @@ def reserve_pallet(self, pallet_name, picking=None):
         else:
             self.write({"u_last_reserved_pallet_name": pallet_name})
 
+    def check_same_picking_priority(self, pickings, mode="mobile"):
+        """Checks if pickings priorities matches with batch priority
+
+        Args:
+            pickings (stock.picking): set of Picking objects
+        Return:
+            Boolean: Returns False if picking priority is different than batch else True
```

Comment:
Technically this function doesn't return a bool

Reply:
Good pick. I changed the method but forgot to change the description. It returns the list of picking priority name

Reply:
done